config/vufind/contentsecuritypolicy.ini

```diff
@@ -47,6 +47,8 @@ worker-src[] = "blob:"
 style-src[] = "'self'"
 style-src[] = "'unsafe-inline'"
 img-src[] = "'self'"
+; If you are using LibGuidesProfile recommendation module, uncomment the line below
+;img-src[] = libapps.s3.amazonaws.com
```

Comment:
I notice most of the img-src here start with 'https:'.  A protocol is not required by the CSP definition, but I understand the intent to avoid loading from an insecure (http) source.  That said when I run a test instance of VuFind it's http so it would be annoying to not support that.  The actual URL loaded (coming from the API data) omits the protocol also (i.e. starts with '//') so it picks up https if the page already uses it.